src/test/java/com/company/models/FixtureStatus.java

```diff
@@ -0,0 +1,188 @@
+package com.company.models;
+
+import twitter4j.*;
+
+import java.util.Date;
+
+public class FixtureStatus implements Status {
```

Comment:
There is a TwitterObjectFactory or something similar that you could use to create status's possibly